README.md

```diff
@@ -1,5 +1,5 @@
 # React Pagination
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_pagination/)
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://DariaVeretyak.github.io/react_pagination/)
```

Comment:
<img width="874" alt="image" src="https://user-images.githubusercontent.com/38065883/173064175-56f58af0-4797-4c74-9198-a266c2af60ef.png">
Looks like for now there is no ability to change `total` value. Please add it